pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -262,6 +296,13 @@ func (m *qosContainerManagerImpl) UpdateCgroups() error {
 		return err
 	}
 
+	// update the qos level cgroup settings for huge pages (ensure they remain unbounded)
```

Comment:
Could this lead to the overcommit for hugepage at the node level?

Reply:
No.  We enforce via validation of pod spec that over-commit is not allowed.